docs/developer-updates/release-notes-v28.md

```diff
@@ -1,24 +1,21 @@
-## ⬆️ Install or Upgrade
+## ⬆️  Install or Upgrade
```

Comment:
how come there are two spaces now? Just better formatting?

Reply:
Yeah not sure, I will fix it back, tnx